src/features/empty-base-layer/less/emptyBaseLayer.less

```diff
@@ -0,0 +1,9 @@
+@import '../../../less/variables';
+
+.ui-grid-viewport .ui-grid-empty-base-layer-container {
+  position: absolute;
+  height: 100% !important;
```

Comment:
The reason for using is because the same div inherits from 'ui-grid-canvas' as well, which gets a dynamic height/width, but I always wanted this div to take the full height of the viewport. So I should be able to find a different way.